src/pages/docs/degen/index.md

```diff
@@ -123,10 +129,10 @@ This command is not only used to find your wallet address, but it also serves as
 #### Verifying Your Wallet's Balance
 
 ```bash
-yagna payment status --network=polygon --driver=erc20
+yagna payment status --network=goerli --driver=erc20
```

Comment:
I believe yagna payment status returns goerli by default. I would prefer to use the simplest command as possible

Reply:
just in case we need to advise participants to switch to holesky I decided to keep this entry, but indeed it could be removed